### PR TITLE
do not collapse console errors on mouseUp

### LINKED ISSUE #192: Click in an expanded exception in the Console should not collapse it
**Description**

If I click anywhere in an expanded exception, it minimizes the exception.
I feel like it should only expand/collapse if you click the caret on the left.

**Steps to reproduce**
1. Type any command resulting in an exception in the console, i.e. `test`
2. Click on the red collapsed exception text to expand it
3. Click anywhere in the expanded exception text

**Expected results**

The exception stays expanded.

**Actual results**

The exception gets collapsed.

### PR DESCRIPTION
**Pull request by spasovski**
fixes #192 

This was clearly intentionally allowed before but I agree the new behaviour is better.

### CODE REVIEW

**Review by mofojed — CHANGES_REQUESTED** [review]
Don reviewed this for UX? You can't select the first line of the exception, we need to be able to select the full exception.

**Comment by dsmmcken** [comment]
true, not sure it's any better, it just shifted the problem from the body to the button.

**Comment by spasovski** [comment]
The button text is selectable (though not easily) - I'd float a copy to clipboard button to the far right of the error expand button and we can manually write the full text to the clipboard.

**Comment by mofojed** [comment]
You can't select the button/top line of text at all right now? A floating copy button might be handy for copying a particular exception, but you should also just be able to copy the console output in general (eg. if you have a bunch of logging lines, then an exception, more logging, you should be able to select all that).

File: packages/console/src/console-history/ConsoleHistoryResultErrorMessage.jsx [hunk 1]
```diff
@@ -112,16 +108,18 @@ class ConsoleHistoryResultErrorMessage extends PureComponent {
...
-          tabIndex="0"
-          className="error-content"
-          onKeyPress={this.handleKeyPress}
-          onMouseDown={this.handleMouseDown}
-          onMouseMove={this.handleMouseMove}
-          onMouseUp={this.handleMouseUp}
-        >
-          {message}
+        <div className="error-content">
+          <button
```
Comment (mofojed) [inline packages/console/src/console-history/ConsoleHistoryResultErrorMessage.jsx:112]: Keep this in a div so you can select text.

Check behaviour in Firefox to ensure right-click does not trigger.

**Review by mofojed — CHANGES_REQUESTED** [review]

File: packages/console/src/console-history/ConsoleHistoryResultErrorMessage.jsx [hunk 2]
```diff
@@ -99,7 +113,7 @@ class ConsoleHistoryResultErrorMessage extends PureComponent {
             <button
               type="button"
               onClick={this.handleToggleError}
-              className="error-btn-link"
+              className={arrowBtnClasses}
```
Comment (mofojed) [inline packages/console/src/console-history/ConsoleHistoryResultErrorMessage.jsx:116]: This looks much better! Only issue is that the button doesn't highlight when you are hovering over it itself.

Could add this here:
```
            onMouseEnter={this.handleMouseEnter}
            onMouseLeave={this.handleMouseLeave}
```
Or just add a `error-message.btn:hover` styling as well

**Review by mofojed — CHANGES_REQUESTED** [review]

**Review by mofojed — APPROVED** [review]

**Review by mofojed — APPROVED** [review]

**Pull request merged**